src/components/NFTPortfolio/Stat.vue

```diff
@@ -47,6 +47,23 @@ export default {
         0
       );
     },
+    collectedClassIds() {
+      const classIdSet = new Set(this.collectedItems.map(n => n.classId));
+      return Array.from(classIdSet);
+    },
+  },
+  watch: {
+    collectedClassIds() {
+      this.lazyGetAllCollectedPurchaseInfo();
+    },
+  },
+  methods: {
+    ...mapActions(['lazyGetNFTPurchaseInfo']),
+    lazyGetAllCollectedPurchaseInfo() {
+      this.collectedClassIds.forEach(classId =>
```

Comment:
Set has forEach, do we really need to convert it to array in `collectedClassIds`?